### ISSUE TITLE
Can you release  pre-training code of CodeT5?

### ISSUE THREAD

**Issue by openRiemann** [salesforce/CodeT5#40]
Hi!
I would like to know how methods like **MIP**(Masked Identifier Prediction), **IT**(Identifier Tagging), **BDG**(Bimodal Dual Generation) etc are implemented. I'd appreciate it if you release pre-training code of CodeT5.

**Comment by yuewang-cuhk** [comment]
你好，你的邮件我已收到，我会尽快查看~~~

**Comment by yuewang-cuhk** [comment]
Hi, we do not have a plan to release the pretraining code so far.
